src/config.rs

```diff
@@ -66,8 +66,8 @@ impl Config {
         let config_dir_path = local_dir.join("cloudy").join("roseate");
 
         if !config_dir_path.exists() {
-            fs::create_dir_all(config_dir_path.clone())?;
-            if let Err(err) = fs::create_dir_all(config_dir_path.clone()) {
+            fs::create_dir_all(&config_dir_path)?;
+            if let Err(err) = fs::create_dir_all(&config_dir_path) {
```

Comment:
![image](https://github.com/user-attachments/assets/df70d885-17fb-4aa0-964d-a5ef94dae9f5)

Reply:
Don't make me remove Arc<T> from the `image_bytes` field in the `Image` struct. (everywhere `image.clone()` is used the entire image loaded in memory will duplicate and WE'VE USED IMAGE.CLONE LIKE EVERYWHERE! 💀)